talk/owt/sdk/base/win/msdkvideoencoder.cc

```diff
@@ -182,9 +182,9 @@ int MSDKVideoEncoder::InitEncodeOnEncoderThread(
   MSDKConvertFrameRate(30, &m_mfxEncParams.mfx.FrameInfo.FrameRateExtN,
                        &m_mfxEncParams.mfx.FrameInfo.FrameRateExtD);
   m_mfxEncParams.mfx.EncodedOrder = 0;
-  m_mfxEncParams.mfx.LowPower = MFX_CODINGOPTION_ON;
+  // m_mfxEncParams.mfx.LowPower = MFX_CODINGOPTION_ON;
```

Comment:
If you don't need this code, please remove it.

Reply:
Ideally we want to keep it. But seems not working for hevc in some cases. Keep it here because we want to revisit it to turn it on sometime.